app/controllers/comments_controller.rb

```diff
@@ -7,6 +7,8 @@ def create
     authorize @comment
     if !@comment.save!
       flash[:error] = 'There was an error saving the post. Please try again.' 
+    else
+      @comment.create_activity :comment, owner: current_user, recipient: @commentable
     end
```

Comment:
This is really minor - consider flipping the `if/else` condition so line 8 becomes `if @comment.save!` and line 9 is swapped with line 11. I think this will make these few lines a little more readable, but that may just be me.